### DISCUSSION TITLE
Pimcore\Model\Notification\Service\NotificationService is @internal now?

### DISCUSSION THREAD

**Discussion by jantomicky** [pimcore/pimcore#16586] [Q&A / Support]
The `Pimcore\Model\Notification\Service\NotificationService` service is now marked as internal in Pimcore 11. Why? We are using it for dispatching notifications to users; what is the recommended practice now?

**Comment by kingjia90** [comment]
Actually it was since [v10](https://github.com/pimcore/pimcore/blob/v10.0.0/models/Notification/Service/NotificationService.php) by https://github.com/pimcore/pimcore/pull/8899, i don't know specifically the reason behind this change, there's no mention in change logs and [docs](https://pimcore.com/docs/platform/Pimcore/Tools_and_Features/Notifications/) still recommends it , maybe it's a mistake. @brusch wdyt?
